web/src/stream_pill.ts

```diff
@@ -70,10 +70,14 @@ export function get_user_ids(pill_widget: StreamPillWidget): number[] {
     return user_ids;
 }
 
-export function append_stream(stream: StreamSubscription, pill_widget: StreamPillWidget): void {
+export function append_stream(
+    stream: StreamSubscription,
+    pill_widget: StreamPillWidget,
+    display_value: (stream: StreamSubscription) => string,
+): void {
```

Comment:
Is using a parameter, `show_stream_sub_size`, a better approach than passing a function?

The `show_stream_sub_size` parameter approach has been employed in a similar PR, #24580.